applications/Profond.md

```diff
@@ -0,0 +1,154 @@
+
+### Team Code Repos
+
+- [https://github.com/emarai](https://github.com/emarai)
+- [https://github.com/ekkirinaldi](https://github.com/orgs/profondHQ/people/ekkirinaldi)
+- [https://github.com/bagasprakasa](https://github.com/bagasprakasa)
+- [https://github.com/internnos](https://github.com/internnos)
+- [https://github.com/adeyusupp](https://github.com/adeyusupp)
+
+### Team LinkedIn Profiles (if available)
+
+- [https://www.linkedin.com/in/adhigunamahendra/](https://www.linkedin.com/in/adhigunamahendra/)
+- [https://www.linkedin.com/in/bagasprakasa/](https://www.linkedin.com/in/bagasprakasa/)
+- [https://www.linkedin.com/in/irfianto/](https://www.linkedin.com/in/irfianto/)
+- [https://www.linkedin.com/in/amajidsinar/](https://www.linkedin.com/in/amajidsinar/)
+- [https://www.linkedin.com/in/ekkirinaldi/](https://www.linkedin.com/in/ekkirinaldi/)
+- [https://www.linkedin.com/in/vincentsalaka/](https://www.linkedin.com/in/vincentsalaka/)
+- [https://www.linkedin.com/in/adeyusupp/](https://www.linkedin.com/in/adeyusupp/)
+
+## Development Status :open_book:
+
+PoC → Here’s our current MVP in Sui [http://console.profond.ai/](http://console.profond.ai/)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
+- **Full-Time Equivalent (FTE):** 2 FTE
+- **Total Costs:** $10.000
+
+### Milestone 1 — No code smart contract builder
+
+- **Estimated duration:** 1 month
+- **FTE:** 2
+- **Costs:** 4,000 USD
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide documentation page on how to create NFT (PSP34) and FT (PSP22) smart contract with our interface. API documentation for project settings will also be provided.  |
+| 0c. | Testing and Testing Guide | All functions will be covered with full integration test, testing guide will also be provided. |
+| 0d. | Article | We will publish an article that explains the no code builder |
+| 1. | No Code NFT Creation (ink!) | Allows users to create NFT collection smart contract (PSP34) using ink! in one of WASM supported Parachains. |
+| 2. | No Code Fungible Token Creation (ink!) | Allows users to create Fungible Token smart contract (PSP22) using ink! in one of WASM supported Parachains. |
+| 3. | Project Management Module | Allows users to create more than one projects |
+| 4. | Multichain Integration | Allows users to choose between Polkadot Parachains, Kusama, or Sui |
+
+### Milestone 2 Example — Analytics and API
+
+- **Estimated Duration:** 1 month
+- **FTE:** 2
+- **Costs:** 6,000 USD
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| 0. | API Documentation | We will provide documentation page on how to create basic NFT collection (PSP34) using our no-code builder and creating a fungible token (PSP22). |
+| 1. | Analytic Functionality | Add on-chain analytic for the smart contract. |
```

Comment:
What exactly does "on-chain analytic" mean? Can you elaborate on this?

Reply:
It is data analytics for each contract in the project. Users can create a new contract on Profond or put in existing contracts.
Each contract created in Profond will come with analytics. Here's an example of on-chain data analytic for marketplace.paras.near. We're using Metabase for now. We will put this on the proposal.

<img width="1361" alt="CleanShot 2023-04-21 at 13 27 18@2x" src="https://user-images.githubusercontent.com/22832454/233557851-b37cb5ab-f740-4510-8250-d23ecba1545e.png">
<img width="1360" alt="CleanShot 2023-04-21 at 13 27 59@2x" src="https://user-images.githubusercontent.com/22832454/233557890-f3d849dd-b89c-474a-8cf0-61e804d58eef.png">